@AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mfdConstants.h

```diff
@@ -164,7 +164,8 @@
 #define MFD_TEXT_IND_WPN_MSL_IMAGE_LINE_2 19
 
 //TSD Root page indices
-#define MFD_IND_TSD_ROOT_SHOW_WPT_DATA_CURRTE 0
+#define MFD_IND_TSD_ROOT_SHOW_WPT_DATA_CURRTE 
+#define MFD_IND_TSD_ROOT_SHOW_SCALE 10
```

Comment:
Why was the number 10 picked here, currently for TSD_ROOT we have only got a 0

Reply:
Talking to Brad though this does need to be independent of variant, but I do not want a point for just a single popout. Could we have a

```cpp
#define MFD_IND_TSD_OPEN_POPOUT 13
```

Reply:
fixed and done